python/python/tests/test_async_client.py

```diff
@@ -1651,7 +1651,8 @@ async def test_zremrangebylex(self, redis_client: TRedisClient):
             )
             == 2
         )
-        assert await redis_client.zrange_withscores(key1, range) == {"a": 1.0, "d": 4.0}
+        zrange_res = await redis_client.zrange_withscores(key1, range)
```

Comment:
In all those places where you store the command output, I would've rename it to the tested command name
zrange_res -> zremrangebylex_res